### PR TITLE
Adding new cryptominer detection

### PR DESCRIPTION
**Pull request by kkadosh**
#1821
 
Add Coin-have and Minero detection.

For coinhave, it can be tested here : [show-online.tv](url)

### CODE REVIEW

File: src/apps.json [hunk 1]
```diff
@@ -5394,6 +5402,17 @@
 			"icon": "Milligram.png",
 			"website": "http://milligram.github.io"
 		},
+		"Minero": {
+			"cats": [
+				"56"
+			],
+			"script": [
```
Comment (jvoisin) [inline src/apps.json]: How about `"//minero\\.cc/lib/minero(?:-miner|-hidden)?\\.min\\.js",` instead?

File: src/apps.json [hunk 2]
```diff
@@ -5394,6 +5402,17 @@
...
+				"https?://minero\\.cc/lib/minero\\.min\\.js",
+				"https?://minero\\.cc/lib/minero-miner\\.min\\.js",
+				"https?://minero\\.cc/lib/minero-hidden\\.min\\.js"
+			],
```
Comment (jvoisin) [inline src/apps.json]: Have you checked if there is a way to detect this via `env` too?

**Comment by AliasIO** [comment]
Merged and updated manually, thank you.

File: src/apps.json (hunk 2 above)
Comment (gadcam) [reply src/apps.json]: There seems to be one : 
```js
"js": {
  "Minero": ""
}
```

**Pull request closed without merging**